### ISSUE TITLE
Is there a limitation of how many header TocBot can display ?

### ISSUE THREAD

**Issue by Nemzag** [tscanlin/tocbot#104]
Hi,
first sorry pro my English, I'm Albanian and I live in francophone part of Belgium…
I downloaded your script, to use it in my personal website, 
and I have a very old files written in 2007, with many headers, with many sub‑headers,
I configured the script to show all headers like this "headingSelector: 'h1, h2, h3, h4, h5, h6',".
But my table of contents show only 37 headers, but doesn't show the rest. 
So my question is does TocBot script have a limitation of how many header he can display in table of contents ?
Because if I remove the last header displayed (in the HTML code), the next is shown in table of content but not the rest.
How can I display all my document headers with your cool script ?
Thank you. And take care.

**Comment by tscanlin** [comment]
Tocbot doesn't have any limits. But maybe they aren't all showing on the screen because the `div` needs to be scrollable? What happens if you give it the style `overflow-y: auto`?

A link or code example would help me better understand the issue so I can help you more easily.

**Comment by Nemzag** [comment]
Hi, thank you pro your quick response, 
first I need to tell you that after downloading your files, I had to change a little bit the CSS, because in the beginning, your style, shown a menu, who had problem in my Firefox navigator, when clicking on it it expanded itself to reduce itself directly, I removed this behavior, to have a complete static menu using the same background-color than the theme is use with prism.js & prism.css (syntaxic coloration) in HTML page.
"overflow-y: auto" is set in CSS... I don't understand what you mean.
I send you the file you asked pro.
[HTML.zip](https://github.com/tscanlin/tocbot/files/2438596/HTML.zip)
Thank pro your help.
